flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveLookupTableSource.java

```diff
@@ -323,5 +324,28 @@ public HiveTablePartitionFetcherContext(
                 }
             }
         }
+
+        @Override
+        public HiveTablePartition toHiveTablePartition(HiveTablePartition partition) {
+            return partition;
+        }
+
+        @Override
+        public ObjectPath getTablePath() {
```

Comment:
I think we need a `HivePartitionContext` which is used by look up source.
And have a `HiveContinuousPartitionContext` extends `HivePartitionContext` which is used by continuous source.

Reply:
So here don't need provides ugly methods to lookup source.